### PR TITLE
Add relevant methods and Javadoc for ActivityCreateCommand

### LINKED ISSUE #42: add ActivityCreateCommand class
To take in user input and create an Activity object as specified by the parsed user input.

### PR DESCRIPTION
**Pull request by ivanaitzliddat**
Add prepare method in ActivityCreateCommand class
Add Javadocs for all methods in ActivityCreateCommand class.

Fixes #42

### CODE REVIEW

File: src/main/java/seedu/splitlah/command/ActivityCreateCommand.java [hunk 1]
```diff
@@ -65,71 +131,144 @@ public void run(Manager manager) {
...
     }
 
+    /**
+     * Returns a double representing the total cost of the activity
+     * by summing up the costs owed by each person involved in the activity.
+     *
+     * @param costList The costs owed by each person involved in the activity.
+     * @return A double representing the total cost of the activity.
+     */
     private static double getTotalCost(double[] costList) {
```
Comment (froststein) [inline src/main/java/seedu/splitlah/command/ActivityCreateCommand.java:225]: This naming could be misleading as totalcost is an attribute, perhaps renaming this function to calculateTotalCost or sumUpTotalCost.

**Review by froststein — CHANGES_REQUESTED** [review]
Overall looks good, just need to change one of the functions name to avoid confusion.

**Review by froststein — APPROVED** [review]
Looks good for merge

**Review by lelerer — APPROVED** [review]
Looks good to merge.

**Pull request merged**